data/saas/config/auth0_config.yml

```diff
@@ -0,0 +1,60 @@
+saas_config:
+    fides_key: auth0_connector_example
+    name: Auth0 SaaS Config
+    type: auth0
+    description: A sample schema representing the Auth0 connector for Fidesops
+    version: 0.0.1
+
+    connector_params:
+      - name: domain
+      - name: client_id
+      - name: access_token
+
+    client_config:
+      protocol: https
+      host: <domain>
+      authentication:
+          strategy: bearer
+          configuration:
+            token: <access_token>
+
+    test_request:
+      method: GET
+      path: /api/v2/users-by-email?email=test@test.com
```

Comment:
Although this would work, we should define the query parameters using the `query_params` property.

```
path: /api/v2/users-by-email
query_params:
  - name: email
    value: test@test.com
```